cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -729,6 +769,64 @@ class SimpleArray
     value_type * m_body = nullptr;
 }; /* end class SimpleArray */
 
+template <typename T>
+void SimpleArray<T>::take_along_axis(SimpleArray<uint64_t> const & indices)
+{
+    if (indices.ndim() != 1)
+    {
+        throw std::runtime_error(Formatter() << "SimpleArray: sorting is only supported in 1D array. "
```

Comment:
Please reword to be like `SimpleArray::take_along_axis() supports only 1D array but the array is N dimension` (replacing N as the variable).